src/ast.hpp

```diff
@@ -218,8 +218,8 @@ namespace Sass {
   public:
     Vectorized(size_t s = 0) : hash_(0)
     { elements_.reserve(s); }
-    Vectorized(const std::vector<T>& vec) :
-      elements_(vec),
+    Vectorized(const std::vector<T> vec) :
```

Comment:
Copy-elision won't work with `const`.
There is little reason to mark arguments passed by value as `const` anyway.

Reply:
OK, adjusted a third time